src/notebook/cells/editor.ts

```diff
@@ -416,7 +415,7 @@ class CodeCellEditorWidget extends CodeEditorWidget implements ICellEditorWidget
       model.source = newValue;
     }
     this.textChanged.emit({
-      line, ch, chHeight, chWidth, coords, position, oldValue, newValue
+      line, ch, chHeight, chWidth, coords, position, newValue
```

Comment:
I had wondered about this on first writing the `ObservableString` interface. Was there a reason for the `oldValue` entry, and if so, should we change the `ObservableString.IChangedArgs` to include it?

Ultimately, this could mean more expensive signals to create and pass around if the cell text is large.